cmake/unittest.cmake

```diff
@@ -26,9 +27,11 @@ add_whole_archive_flag(onnx tmp)
 list(APPEND ${UT_NAME}_libs ${tmp})
 list(APPEND ${UT_NAME}_libs onnx_proto)
 list(APPEND ${UT_NAME}_libs onnxifi_loader)
+list(APPEND ${UT_NAME}_libs onnxifi)
```

Comment:
Better to separate the dependencies of onnxifi test driver from normal unittests (aka create another ${ONNXIFI_TEST_DRIVER}_libs instead of mixing it into ${UT_NAME}_libs.